default.nix

```diff
@@ -21,24 +21,39 @@ let
       ];
   });
 
-  haskellPackages = reflex-platform.ghcjs.override {
-    overrides = self: super: {
-      hpc = self.callHackage "hpc" "0.6.0.3" { };
-      ghc-lib-parser = pkgs.haskell.lib.dontHaddock
-        (super.callHackage "ghc-lib-parser" "8.8.1" { });
-      ghc-syntax-highlighter = pkgs.haskell.lib.dontCheck
-        (self.callCabal2nix "ghc-syntax-highlighter" (pkgs.fetchFromGitHub {
-           owner = "neongreen";
-           repo = "ghc-syntax-highlighter";
-           sha256 = "1d7d0diiqilw30d84cws8zqm36cc101qy2mxxbixbls5qd3kyk8a";
-           rev = "8c47d4c04a72464fabf8fa55c6e7de6b54c22e8a";
-        }) { });
-      ormolu = pkgs.haskell.lib.dontCheck
-        (self.callCabal2nix "ormolu" ormolu { });
-      ormolu-live = optimize (pkgs.haskell.lib.dontCheck
-        (self.callCabal2nix "ormolu-live" source { }));
-    };
+  overrides = self: super: let
+    whenGhcjs = o: d: if self.ghc.isGhcjs or false then (o d) else d;
+  in {
+    hpc = self.callHackage "hpc" "0.6.0.3" { };
+    ghc-lib-parser = pkgs.haskell.lib.dontHaddock
+      (super.callHackage "ghc-lib-parser" "8.8.1" { });
+    ghc-syntax-highlighter = pkgs.haskell.lib.dontCheck
+      (self.callCabal2nix "ghc-syntax-highlighter" (pkgs.fetchFromGitHub {
+          owner = "neongreen";
+          repo = "ghc-syntax-highlighter";
+          sha256 = "1d7d0diiqilw30d84cws8zqm36cc101qy2mxxbixbls5qd3kyk8a";
+          rev = "8c47d4c04a72464fabf8fa55c6e7de6b54c22e8a";
+      }) { });
+    ormolu = pkgs.haskell.lib.dontCheck
+      (self.callCabal2nix "ormolu" ormolu { });
+    ormolu-live = whenGhcjs optimize (pkgs.haskell.lib.dontCheck super.ormolu-live);
+    hlint = pkgs.haskell.lib.doJailbreak super.hlint;
+    reflex-dom-core = pkgs.haskell.lib.dontCheck super.reflex-dom-core;
   };
 
-in
-  haskellPackages.ormolu-live
+in (reflex-platform).project ({ pkgs, ... }: {
```

Comment:
Is `(reflex-platform)` different from `reflex-platform`?

Reply:
Nope, they're the same.

Reply:
Updated.